dev/AppLifecycle/AppInstance.cpp

```diff
@@ -33,30 +34,37 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
         // Push past the '----' commandline argument prefix.
         argsStart += 4;
```

Comment:
Pre-existing bug: I can perform command line injection by creating a file called "C:\----ms-protocol" with an alternate stream named "attack-you-here .txt". Launching that file will pass the command line "C:\----ms-protocol::attack-you-here .txt" which will get mis-parsed as a ms-protocol launch with URL "attack-you-here" (which can be replaced with any desired attack string).

Reply:
Valid concern but belongs to AppLifecycle (not PushNotifications).

@oldnewthing please file a Bug ([New Issue](https://github.com/microsoft/ProjectReunion/issues)) and assign to @aeloros for follow up. Thank you

Reply:
Unless [Handle '----' better and enforce having a space before it. #907](https://github.com/microsoft/ProjectReunion/issues/907) is that bug?

Describes the bug but not the security/attack-vector angle? Add some more color commentary to that bug instead of a new one?

Reply:
Added color commentary to existing bug.